src/vs/platform/environment/node/argv.ts

```diff
@@ -15,76 +15,86 @@ import { MIN_MAX_MEMORY_SIZE_MB } from 'vs/platform/files/common/files';
 
 const options: minimist.Opts = {
 	string: [
-		'locale',
-		'user-data-dir',
-		'extensions-dir',
-		'folder-uri',
-		'extensionDevelopmentPath',
-		'extensionTestsPath',
-		'install-extension',
-		'disable-extension',
-		'uninstall-extension',
+		'database',
```

Comment:
Please add `// {{SQL CARBON EDIT}}` tags in this file as well.  And limit unnecessary edit when possible.